sdk/vision/ci.yml

```diff
@@ -48,7 +48,6 @@ extends:
   parameters:
     ServiceDirectory: vision
     EnableBatchRelease: true
-    EnableVerifyTypeSpecCodeGeneration: false
```

Comment:
This is a temp change just to see the ADO task fail. Will be reverted in future commits to this PR